CRSP-Tests.postman_collection.json

```diff
@@ -174,9 +282,83 @@
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "https://frmfaas.sybrin.com/function/off-channel-router-setup-processor/execute",
```

Comment:
please make all urls internal

Reply:
I didnt save them when i changed the URLS , changes saved and redeployed .